.license-scan-overrides.jsonl

```diff
@@ -1,4 +1,5 @@
 {"name": "github.com/chzyer/logex", "licenceType": "MIT"}
+{"name": "github.com/grpc-ecosystem/go-grpc-middleware/v2", "licenceType": "Apache-2.0"}
```

Comment:
FYI: Not sure why this only shows up now, but running latest `go-makefile-maker` also produced this diff, so I guess we need this. 🤷